README.md

```diff
@@ -33,5 +33,7 @@ Contributing to Capacitor may involve writing TypeScript, Swift/Objective-C, Jav
 
 Please read the [Contributing](.github/CONTRIBUTING.md) guide for more information.
 
+For details on updating the Capacitor website or documentation, [see here](.github/site/CONTRIBUTING.md).
```

Comment:
is this the correct way to link?

Reply:
no, unless you create a `site` folder inside `.github` folder and move the `CONTRIBUTING.md` there

Reply:
updated